src/libstd/collections/hash/table.rs

```diff
@@ -835,18 +838,27 @@ pub struct IterMut<'a, K: 'a, V: 'a> {
     elems_left: usize,
 }
 
+unsafe impl<'a, K: Sync, V: Sync> Sync for IterMut<'a, K, V> {}
+unsafe impl<'a, K: Sync, V: Send> Send for IterMut<'a, K, V> {}
```

Comment:
I'm unsure. See https://github.com/rust-lang/rust/pull/27615#discussion_r36679199 . We're also discussing it on IRC at the moment:

- https://botbot.me/mozilla/rust-libs/msg/46882011/
- https://botbot.me/mozilla/rust-internals/msg/46880251/

I think the conclusion is that both `K: Send` and `K: Sync` are correct, and that `K: Send` is the more useful bound (i.e. there are more things that are `Send` but not `Sync` than visa versa).